conda-env-mod

```diff
@@ -586,14 +596,41 @@ if [ "${ENV_PREFIX}" != "" ]; then
 fi
 
 # Logic for environment name and location
-if [ "${ENV_PREFIX}" != "" ]; then
-	CONDA_ARGS+=( --prefix "${ENV_PREFIX}" )
-	if [ "${ENV_NAME}" == "" ]; then
-		ENV_NAME="${ENV_PREFIX##*/}"
+# if [ "${ENV_PREFIX}" != "" ]; then
+# 	CONDA_ARGS+=( --prefix "${ENV_PREFIX}" )
+# 	if [ "${ENV_NAME}" == "" ]; then
+# 		ENV_NAME="${ENV_PREFIX##*/}"
+# 	fi
+# fi
+# if [[ "${ENV_NAME}" != "" ]] && [[ "${ENV_PREFIX}" == "" ]]; then
+# 	CONDA_ARGS+=( --name "${ENV_NAME}" )
+# fi
+
+if [ "${ENV_FILE}" != "" ]; then
+	CONDA_ARGS+=( --file "${ENV_FILE}" )
+	read -r line<$ENV_FILE
+	#echo $line
+	Opt_NAME=$(echo "$line"|grep name)
+	if [[ "$Opt_NAME" != "" ]]; then
+		Opt_YAML=1
+		ENV_NAME=$(echo $line| cut -d' ' -f2)
+	else
+		echo "Please specify the name of the new environment."
+		echo -n "Enter Name:"
```

Comment:
all of the optional arguments may be specified in command line. so account for that scenario.